app/middleware/error-handler.js

```diff
@@ -84,6 +85,11 @@ module.exports = function errorHandler (err, req, res, next) {
       service: err.service,
       status: err.statusCode
     })
+  } else if (err instanceof AxiosError) {
+    logger.info(`Unhandled AxiosError caught: ${err.message}`, {
```

Comment:
Curious that we log this at info level…

Reply:
I think it's just because we also send the error to sentry using `Sentry.captureException(err)`. If we logged at error level, we would create 2 events in sentry